android/titanium/src/java/org/appcelerator/titanium/util/TiDownloadManager.java

```diff
@@ -88,7 +88,7 @@ public void download(URI uri, TiDownloadListener listener)
 	 * <p>
 	 * Returns null if failed to download content or if given an invalid argument.
 	 */
-	public InputStream blockingDownload(final URI uri)
+	public InputStream blockingDownload(final URI uri) throws Exception
```

Comment:
We need to update the `@return` comment since it no longer returns `null` if failed to download.